tests/unit/wait.tcl

```diff
@@ -98,6 +98,35 @@ start_server {} {
         $rd close
         $rd2 close
     }
+
+    start_server {} {
+        test {Setup a new replica} {
+            r replicaof $master_host $master_port
+            wait_for_ofs_sync $master r
+            wait_for_ofs_sync $master $slave
+        }
+
+        test {WAIT in script will work} {
+            # Pause the old replica so it can not catch up the offset.
+            pause_process $slave_pid
+
+            # Primary set a new key and wait the new replica catch up the offset.
+            $master set foo bar
+            wait_for_ofs_sync $master r
+
+            # Wait for the new replica to report the acked offset to the primary.
+            # Because the old replica is paused, so the WAIT can only return 1.
+            # In the old code it will return 2, because the fake client's woff is
+            # always 0 so WAIT will count all the replicas.
```

Comment:
Nobody will know what the "old code" is. This is good for the PR description, but in the code comments we need to describe the regression or mention issue number instead.

Also, we have only one replica here so WAIT can only return 1 or 0, never 2, right? Maybe it is a mixup with WAITAOF?

```suggestion
            # Wait for the new replica to report the acked offset to the primary.
            # Because the old replica is paused, the WAIT can only return 0.
            # In an earlier version it returned 1, because the fake client's
            # woff is always 0 so WAIT counted all the replicas.
```

Reply:
The test has two replicas, one is at the front, and the other is what I added now

Reply:
Ah, ok